### DISCUSSION TITLE
Mautic - Open source Marketing Automation Platform

### DISCUSSION THREAD

**Discussion by luckycreationsindia** [coollabsio/coolify#2296] [Service Template Requests]
Hi,

I was searching for opensource marketing tools and came across this "Mautic". It's open source Marketing Automation Platform to manage Leads & Ads for various platform.

Git Repo: https://github.com/mautic/mautic

They have docker image too - https://hub.docker.com/r/mautic/mautic
Docker Image Git- https://github.com/mautic/docker-mautic

**Comment by JuanxCursed** [comment]
@luckycreationsindia 
I've just created a PR for Mautic: 
https://github.com/coollabsio/coolify/pull/2978
